app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -322,10 +322,12 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
     private fun actionSorter(actions: Array<String>): Array<String> {
         val order = hashMapOf<String, Int>()
 
-        order["org.mozilla.geckoview.COPY"] = 0
-        order["CUSTOM_CONTEXT_MENU_SEARCH"] = 1
-        order["org.mozilla.geckoview.SELECT_ALL"] = 2
-        order["CUSTOM_CONTEXT_MENU_SHARE"] = 3
+        order["CUSTOM_CONTEXT_MENU_EMAIL"] = 0
```

Comment:
so basically if email and call are present we want to prioritize those? (and in those cases search is probably not what they want to do anyway)

Reply:
Yes, exactly :)